.github/workflows/codecov.yml

```diff
@@ -12,12 +12,12 @@ jobs:
       - uses: actions/setup-go@v4
         with:
           go-version: 'stable'
-      - name: Install dependencies 📦
+      - name: Install dependencies
         run: go mod tidy
-      - name: Run tests with coverage ✅
+      - name: Run tests with coverage
         run: go test -v -coverprofile="coverage.out" ./...
-      - name: Upload coverage report to Codecov 📝
-        uses: codecov/codecov-action@v4
+      - name: Upload coverage report to Codecov
+        uses: codecov/codecov-action@v3
```

Comment:
Seems codecov delete their v4 release. So I fallback it to v3.

Reply:
So bad.